### ISSUE TITLE
`FastText.wv.word_vec` word vectors with Polish character are different than the Facebook's

### ISSUE THREAD

**Issue by durszlaczek** [RaRe-Technologies/gensim#2276]
#### Description
Facebook's and Gensim's word vectors are different for words with Polish characters.

#### Steps to Reproduce

Example (I'm using Polish common crawl vectors):
```
wget https://s3-us-west-1.amazonaws.com/fasttext-vectors/word-vectors-v2/cc.pl.300.bin.gz 
unzip cc.pl.300.bin.gz 
```

Gensim's (1)
```
from gensim.models import FastText
m = FastText.load_fasttext_format('cc.pl.300.bin')
print(m['głowa'][:10])
```
Facebook's (2)
```
from fastText import load_model
m_fb = load_model('cc.pl.300.bin')
print(m_fb.get_word_vector('głowa')[:10])
```

Oh, and 'głowa' is present in both vocabularies
```
print('głowa' in m.wv.vocab)
print('głowa' in m_fb.get_words())
```

Seems like an encoding issue (?)

#### Expected Results
Values in both vectors (1), (2) are different.

#### Actual Results
Values in  (1), (2) differ.

#### Versions
```
gensim 3.6.0
fasttext 0.8.22
```

**Comment by gojomo** [comment]
Just guessing: maybe one of the libraries is calculating the character n-grams differently/wrongly? (Perhaps, as @durszlaczek speculates, due to an encoding issue.) A 1st step in debugging could be to verify that the raw full-word vector for 'głowa' is identical. If so – and I'd expect it to be – look closer at how subword n-grams might be handled differently.

**Comment by mpenkov** [comment]
I think this may be a duplicate of https://github.com/RaRe-Technologies/gensim/issues/2059

**Comment by menshikh-iv** [comment]
@mpenkov yes, you are right
@durszlaczek thanks for the report, we know about it and I hope we'll fix it soon

Closed, because this is a duplicate of #2059
